site_src/routes/pages.ts

```
@@ -0,0 +1,199 @@
+    } catch (err) {
+      logError('website /me dashboard failed:', err);
+      return c.text('Failed to load dashboard', 500);
+    }
+  });
+
+  app.get('/', (c) => {
+    const user = c.get('user');
+    return c.redirect(user ? '/me' : '/about');
+  });
+
+  app.get('/about', (c) => c.html(AboutPage({ nonce: c.get('nonce'), lv999: c.req.query('lv') === '999', goof: c.req.query('theme') === 'goof', user: navUser(c) }).toString()));
```

ESLint

[error] Expected a line break after this opening brace.

(object-curly-newline)